src/common/error.rs

```diff
@@ -0,0 +1,27 @@
+#![allow(dead_code)]
```

Comment:
I would rather leave the errors in config.rs for now, we might want to revisit this when we work on transitioning away from using anyhow